app/views/modals/create-secret.html

```diff
@@ -0,0 +1,10 @@
+<div class="create-secret-modal">
+  <div class="modal-header">
+    <h2>Create Secret</h2>
+  </div>
+  <div class="modal-body">
+    <ng-include ng-controller="CreateSecretController" src="'views/_create-secret.html'"></ng-include>
```

Comment:
Well we spoke about this with @jwforres and I think the outcome was not to create a directive, but instead a standalone view and controller from which that could be registered in `app.js` for the `/project/:project/create-secret` page. I 'think' thats a cleaner solution but not not 100% sure.. what do you think @spadgett ?